.ci-build.sh

```diff
@@ -2,11 +2,29 @@
 
 echo Entering "$(cd "$(dirname "$0")" && pwd -P)/$(basename "$0")" in `pwd`
 
-# Fail the whole script if any command fails
 set -e
 
 export SHELLOPTS
 
-. ./.ci-build-without-test.sh
```

Comment:
Why not just keep `./.ci-build-without-test.sh`? Isn't `.ci-build.sh` now doing exactly what `./.ci-build-without-test.sh` was doing before?

Reply:
Fixed